news/october 2021 update.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: default
+title: october 2021 update
+center: false
+---
+
+
+## jmcpil and gmcpil
```

Comment:
Everything looks good except this file, It should probably have more detail. Perhaps explaining sound

Reply:
Ok